pgd_splicer/ProcessPDBTask.py

```diff
@@ -755,6 +789,9 @@ def parseWithBioPython(code, props, chains_filter=None):
                 prev       = None
                 if res_id in residues:
                     del residues[res_id]
+                if 'old_id' in vars():
```

Comment:
Let's add a comment about what `vars()` is so future devs and future us knows it's a python thing and not a Django or PGD thing.